shared_testutil/mocknet.go

```diff
@@ -38,25 +38,27 @@ import (
 )
 
 type Libp2pTestData struct {
-	Ctx            context.Context
-	Ds1            datastore.Batching
-	Ds2            datastore.Batching
-	StoredCounter1 *storedcounter.StoredCounter
-	StoredCounter2 *storedcounter.StoredCounter
-	Bs1            bstore.Blockstore
-	Bs2            bstore.Blockstore
-	DagService1    ipldformat.DAGService
-	DagService2    ipldformat.DAGService
-	GraphSync1     graphsync.GraphExchange
-	GraphSync2     graphsync.GraphExchange
-	Loader1        ipld.Loader
-	Loader2        ipld.Loader
-	Storer1        ipld.Storer
-	Storer2        ipld.Storer
-	Host1          host.Host
-	Host2          host.Host
-	AllSelector    ipld.Node
-	OrigBytes      []byte
+	Ctx              context.Context
+	Ds1              datastore.Batching
+	Ds2              datastore.Batching
+	DTStoredCounter1 *storedcounter.StoredCounter
+	DTStoredCounter2 *storedcounter.StoredCounter
+	StoredCounter1   *storedcounter.StoredCounter
```

Comment:
Can this be called DIDStoredCounter1 & 2 like the ones above? or DealStoredCounter1 &2?

Reply:
done